src/cli/account-manager.ts

```
@@ -0,0 +1,350 @@
+  return `CLAUDE_CONFIG_DIR="${toShellPath(expandHome(account.configDirEnv))}" `;
+}
+
+/** Render the accounts.sh contents from a registry object. Pure function. */
+export function generateAccountsSh(reg: Registry): string {
+  const accounts = reg.accounts || [];
+  const defaultLabel = resolveDefaultLabel(reg);
+  const defaultAccount = accounts.find((a) => a.label === defaultLabel);
+  const pad = Math.max(0, ...accounts.map((a) => a.label.length));
+
+  const L: string[] = [];
+  L.push('# CodeV multi-account shell integration — GENERATED FILE (safe to delete).');
```

ESLint

[error] Replace `'#·CodeV·multi-account·shell·integration·—·GENERATED·FILE·(safe·to·delete).'` with `⏎····'#·CodeV·multi-account·shell·integration·—·GENERATED·FILE·(safe·to·delete).',⏎··`

(prettier/prettier)